src/libs/freetype/lv_freetype.c

```diff
@@ -268,9 +226,9 @@ const char * lv_freetype_get_pathname(FTC_FaceID face_id)
  *   STATIC FUNCTIONS
  **********************/
 
-static bool freetype_on_font_create(lv_freetype_font_dsc_t * dsc)
+static bool freetype_on_font_create(lv_freetype_font_dsc_t * dsc, uint32_t max_glyph_cnt)
 {
-    lv_cache_t * glyph_cache = lv_freetype_create_glyph_cache();
+    lv_cache_t * glyph_cache = lv_freetype_create_glyph_cache(max_glyph_cnt * 2);
```

Comment:
Why multiply with 2? A comment would be better here.

Reply:
Commented.